.github/workflows/pr.yml

```diff
@@ -612,7 +612,7 @@ jobs:
           chmod a+x e2e-*
           chmod a+x minikube-*
           START_TIME=$(date -u +%s)
-          KUBECONFIG=$(pwd)/testhome/kubeconfig MINIKUBE_HOME=$(pwd)/testhome ./e2e-linux-amd64 -minikube-start-args=--driver=docker  -test.run "(TestAddons|TestCertOptions)" -test.timeout=10m -test.v -timeout-multiplier=1.5 -binary=./minikube-linux-amd64 2>&1 | tee ./report/testout.txt
```

Comment:
add same s|TestSkaffold to all addon tests in all platforms (virtualbox, docker ... ) there are multiple jobs

and also then copy the whole thing except the first 10 lines to master.yml

Reply:
thanks, done!